docker/owlbot/php/Dockerfile

```diff
@@ -39,6 +39,9 @@ ENV SYNTHTOOL_TEMPLATES="/synthtool/synthtool/gcp/templates"
 ENV npm_config_cache=/tmp/.npm
 RUN mkdir /tmp/.npm && chmod 1777 /tmp/.npm
 
+# Install @prettier/plugin-php
+RUN npm install --global prettier @prettier/plugin-php@^0.18.4
```

Comment:
I might make this `0.8.4` rather than `^0.18.4`.

Reply:
(otherwise it's hard to know specifically what `patch` version we might have gotten).